api/app/src/main/kotlin/packit/model/dto/PacketGroupDisplay.kt

```diff
@@ -0,0 +1,11 @@
+package packit.model.dto
+
+interface PacketGroupDisplay {
+    val displayName: String
+    val packetDescription: String?
```

Comment:
Is this the latest packet description, in the same way the display name is the latest packet display name?  I think just "description" would probably be a more consistent name. 

Also - should latest description be in the packet group table as well as latest display name?

Reply:
Renamed